airflow/serialization/serialized_objects.py

```diff
@@ -265,12 +266,9 @@ class SerializedBaseOperator(BaseOperator, BaseSerialization):
     Class specific attributes used by UI are move to object attributes.
     """
 
-    _decorated_fields = {'executor_config', }
+    _decorated_fields = {'executor_config', 'params'}
```

Comment:
I think adding this change now changes the format of the serialization we store, and since the previous format was in 1.10.7 I thin kthis means we now need to think about adding a v2 of the format and how we deal with upgrade etc.

Reply:
Yup, agree

Reply:
Removed this change